packer/linux/conf/bin/bk-configure-docker.sh

```diff
@@ -42,8 +42,7 @@ if ! docker run \
   --pull=never \
   --rm \
   "tonistiigi/binfmt@${QEMU_BINFMT_DIGEST}" \
-    --install all
-then
+  --install all; then
```

Comment:
This is a bit unfortunate, I would prefer i the formatter did not do this, but I don't consider it important enough to be fatal to the goal of having a formatter.

Luckily, to mitigate potential confusion between the end of the conditional and start of the body of the if statement, most text editors will highlight the `then` with coloured crayon technology.